cypress/e2e/settings/security-user-settings-tab.spec.ts

```diff
@@ -0,0 +1,81 @@
+/*
+Copyright 2023 Suguru Hirahara
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+/// <reference types="cypress" />
+
+import { HomeserverInstance } from "../../plugins/utils/homeserver";
+
+describe("Security user settings tab", () => {
+    let homeserver: HomeserverInstance;
+
+    afterEach(() => {
+        cy.stopHomeserver(homeserver);
+    });
+
+    it("should be rendered properly", () => {
+        cy.startHomeserver("default").then((data) => {
+            homeserver = data;
+            cy.initTestUser(homeserver, "Hanako");
+        });
+
+        cy.openUserSettings("Security");
```

Comment:
This `it`-block does not contain any test (or I don't see it). What is the purpose of it?

Reply:
I have thought `cy.openUserSettings` itself included assertions in it and it was worth checking whether Security tab could be opened properly, but this can be surely removed in favor of the test below, which calls `openUserSettings("Security")`.

I am going to remove this.